ZipBuilder/Template/Firebase.h

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
It's ugly to have another copy of Firebase.h, etc.

Should we consider open sourcing the Firebase pod and associated release process updates in Firebase 6?

Reply:
I agree, that sounds good to me but we'll need to discuss it at length to ensure there are no other issues with it.